### ISSUE TITLE
Mapping library

### ISSUE THREAD

**Issue by danielfdsilva** [NASA-IMPACT/covid-dashboard#557]
As part of the development of the dashboard evolution we're considering moving away from Mapbox.
This is mostly motivated by their licensing model, but also by the fact that we'd like to support different projections.

After some research, Leaflet is mentioned several times as one of the top mapping libraries.
D3 also gets a mention, but it is not really a mapping library - although it has powerful visualization options managing maps would get complicated quickly.

Leaflet is easy to use and supports a wide range of formats and projections, including Vector Tiles (thought a plugin).
It doesn't seem to support data-driven styling the same way that Mapbox does but styles can be applied manually.

In conclusion, I'd say that Leaflet could easily be used here with a tiled basemap and data displayed on top (geojson, vector, more tiles).
If we have to do very heavy data-driven styling with a lot of layers, things get tricky.

@leothomas How complex is the data we have to show? Is it very different than what we're doing now?

**Comment by leothomas** [comment]
For the time being, I've been mostly focused on the downscaled CMIP6 data: raster data with 0.25 x 0.25 degree precision and worldwid extent (land only). Quite similar to the CO2/NO2 tiles for example.

What do you mean by "data-driven styling"?
